x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form.tsx

```diff
@@ -44,10 +45,25 @@ export const PipelineForm: React.FunctionComponent<PipelineFormProps> = ({
 
   const [isTestingPipeline, setIsTestingPipeline] = useState<boolean>(false);
 
+  const processorStateRef = useRef<OnUpdateHandlerArg>();
+
   const handleSave: FormConfig['onSubmit'] = async (formData, isValid) => {
-    if (isValid) {
-      onSave(formData as Pipeline);
+    let override: any = {};
```

Comment:
This makes me nervous. Can it at least be something like `Partial<Pipeline>` or `DeepPartial<Pipeline>`?